antora.yml

```diff
@@ -24,7 +24,7 @@ asciidoc:
     ocp4-ver: "4.12"
     # for the project
     che-plugin-registry-directory: che-plugin-registry
-    devworkspace-operator-index: registry.redhat.io/redhat/redhat-operator-index:v{ocp4-ver}
+    devworkspace-operator-index: quay.io/devfile/devworkspace-operator-index:release
```

Comment:
For the purpose of a restricted install, we probably want `quay.io/devfile/devworkspace-operator-index:release-digest` (the index with image digests). Maybe as a new property (e.g. `{devworkspace-operator-index-digest}`? 

Also, since I assume the `devworkspace-operator-index` property is unchanged in downstream, do we need to update some downstreaming script to reset this to `registry.redhat.io/redhat/redhat-operator-index:v{ocp4-ver}`?

Reply:
Agreed.